sdk/servicebus/microsoft-azure-servicebus/src/test/java/com/microsoft/azure/servicebus/ManagementClientProxyTest.java

```diff
@@ -0,0 +1,57 @@
+package com.microsoft.azure.servicebus;
+
+import com.microsoft.azure.servicebus.management.ManagementClient;
+import com.microsoft.azure.servicebus.management.QueueDescription;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import org.junit.Assert;
+import org.junit.Test;
+import java.io.IOException;
+import java.net.*;
+import java.util.*;
+import java.net.InetSocketAddress;
+import java.net.ProxySelector;
+import java.net.URI;
+import java.util.LinkedList;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+
+public class ManagementClientProxyTest {
+    @Test
+    public void managementClientWithProxy() throws InterruptedException, ServiceBusException {
+        String proxyHostName = "127.0.0.1";
+        int proxyPort = 8888;
+        final ProxySelector systemDefaultSelector = ProxySelector.getDefault();
+
+        ProxySelector.setDefault(new ProxySelector() {
+            @Override
+            public List<Proxy> select(URI uri) {
+                if (uri != null
+                    && uri.getHost() != null
+//                    && uri.getHost().equalsIgnoreCase(proxyHostName)
```

Comment:
I don't quite know why this sample code has this line. Since I am using localhost and the host name will be different from service bus host. If I enable this line, the proxy cannot be created properly. I don't think this line is necessary but just want to point out.